src/Exception/FakeRepositoryException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace SymfonyCasts\Bundle\ResetPassword\Exception;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver   <ryan@symfonycasts.com>
+ */
+final class FakeRepositoryException extends \Exception implements ResetPasswordExceptionInterface
+{
+    public function getReason(): string
+    {
+        return 'Change repository signature for request_password_repository in config/packages/reset_password.yaml.';
```

Comment:
Please updaate the `request_password_repository` configuration in `config/packages/reset_password.yaml` to point to your "request password repository` service.

Reply:
done